src/components/Articles/SingleArticle.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import axiosInstance from "../utilities/axios"
+
+const SingleArticle = () => {
+  const { articleId } = useParams()
+  const [article, setArticle] = useState(null)
+
+  useEffect(() => {
+    const fetchArticle = async () => {
+      try {
+        const response = await axiosInstance.get(`/articles/${articleId}`)
+        setArticle(response.data.article)
+      } catch (error) {
+        console.error(`Error fetching article with ID ${articleId}:`, error)
```

Comment:
Instead of a console log of error, would be best to implement an error state - here are the notes for more info https://notes.northcoders.com/courses/js-front-end/error-handling